boot/nuttx/main.c

```diff
@@ -24,6 +24,7 @@
 #include <nuttx/config.h>
 
 #include <stdio.h>
```

Comment:
Do we need to consider if `#include <stdio.h>` can be removed?

Reply:
Indeed, it is not necessary.